e2e/cypress/integration/onboarding/tutorial_guided_tour_spec.js

```diff
@@ -37,8 +37,8 @@ describe('Onboarding', () => {
             should('be.visible').
             and('contain', 'Send a message');
 
-        // # Close the posting messages tip
-        cy.get(selectors.tipNext).click();
```

Comment:
2/5 Not sure if we should remove the click test, instead we should add tests for both, click and enter keypress event.